rand_distr/README.md

```diff
@@ -11,9 +11,10 @@
 Implements a full suite of random number distributions sampling routines.
 
 This crate is a super-set of the [rand::distributions] module, including support
-for sampling from Beta, Cauchy, ChiSquared, Dirichlet, exponential, Fisher F,
-Gamma, Log-normal, Normal, Pareto, Poisson, StudentT, Triangular, Circle,
-Sphere and Weibull distributions.
+for sampling from Beta, Binomial, Cauchy, CircleChiSquared, Dirichlet, exponential,
```

Comment:
Should this be `Circle, ChiSquared`? Did you remove the `Sphere` intentionally?

Reply:
The sphere is mentioned... aha, didn't delete enough here.